### DISCUSSION TITLE
🧩 FEATURE REQUEST:  VS Code Extension

### DISCUSSION THREAD

**Discussion by udayax** [dailydotdev/daily#616] [Feature Requests]
## Feature Request
To create VS Code Extension for app.daily.dev page

### Description
Is it's possible to create VS Code Extention for https://app.daily.dev/?
As we developers spend lot of time in IDEs, 
It will be much helpful if feeds are shown in vs code itself.
We can make use of the native power of vs code extension library ( Notifications, Storage, etc..)

### Solution
Can create a custom vs code extension for feed page with help of Native VS code library like [marquee ](https://marquee.activecove.com/)extension in vs code and show articles in a new tab within vs code using extensions web view.

### Alternate Solutions
If it's not possible to create a native vs code extension, we can just show https://app.daily.dev/ page just using web view in the extension.

**_Thank you for making the developer's life easier 🤍_**

**Comment by nimrodkra** [comment]
Hi @udaya28, awesome idea! Thank you :) At the moment, our team is very much focused on our browser extension and web app. We do plan to expand to additional platforms in the future; however, we can't commit to a specific timeline at the moment. Once we will have enough resources to expand, we will definitely consider this idea 🙌

**Comment by udayax** [comment]
Hi @nimrodkra,  
Thanks for considering :)
I am happy to contribute in the future ✨
